### ISSUE TITLE
Participated projects visible for all on user profile

### ISSUE THREAD

**Issue by Swetlana1** [liqd/a4-meinberlin#530]
If I write a comment on platform meinBerlin and take part in a participation process, everyone can see my profile and in which participation process I took part, even I'm not logged in. It's not anonymous enough.

**Comment by xi** [comment]
- In which project a user has participated, is intentionally public information. Even if it is not displayed on the user profile it can still be found by scraping the site. Still it may not be the best idea to aggregate this information in one place.
- This list of projects was meant as a placeholder until we can show more relevant information (e.g. an event feed). We need to re-evaluate these options if displaying aggregated information is a concern.
- We simply have nothing to show on the profile page. Once we remove the list, only the username will be left. We might add manually added content like a bio or profile picture. However, most users will not change those setting, resulting in very much useless profile pages.

In summary: We need to ask ourselves whether we want to have a profile page in the first place.
